students/5/modern-js.js

```diff
@@ -0,0 +1,35 @@
+//
+// File: modern-js.js
+// Date: 5/30/2022
+// Desc: Modernize this legacy JavaScript:
+//       - Replace ALL functions with arrow functions (this includes the sort callback function)
+//       - Replace var with let or const.
+//
+
+const displayResults = (personList, count) => {
```

Comment:
Does count really need to be in this function signature?

Reply:
issue resolved